app/packs/src/decidim/friendly_signup/lib/instant_validator.js

```diff
@@ -4,14 +4,12 @@
 // compatible with abide classes https://get.foundation/sites/docs/abide.html
 export default class InstantValidator {
   // ms before xhr check
-  static get TIMEOUT() {
-    return 150;
-  }
 
   constructor($form) {
     this.$form = $form;
     this.$inputs = $form.find("[data-instant-attribute]");
     this.url = this.$form.data("validationUrl");
+    this.TIMEOUT = null;
```

Comment:
I think we should keep the static variable `TIMEOUT` to define a timeout period as follow : 
```
 static get TIMEOUT() {
    return 600;
  }
```

However we can define a new variable in constructor `this.timeoutID = null;` and store the `setTimeout` ID to avoid stacking timeouts